src/components/CheckboxInput/CheckboxInput.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { v4 as uuid } from 'uuid';
+import './CheckboxInput.scss';
+
+const CheckboxInput = ({
+  id,
+  className,
+  selected,
+  onChange,
+  isDisabled,
+  children,
+}) => {
+  const [isSelected, setIsSelected] = useState(selected || false);
+  const inputId = id || uuid();
+
+  const handleChange = () => {
+    setIsSelected(!isSelected);
+    onChange(!isSelected);
+  };
+
+  return (
+    <div className={cx('checkboxInput', { isDisabled })}>
+      <input
+        id={inputId}
+        checked={isSelected}
+        disabled={isDisabled}
+        onChange={handleChange}
+        type="checkbox"
+        className={className}
+        value={isSelected}
+        style={{ width: '24px', height: '24px' }}
+      />
+      <span className="checkboxInputInstructions">{children}</span>
```

Comment:
Whad do we think about making this a label?

Reply:
See reply to @nathanyoung 's comment.